plutus-tutorial/markdown/2019-04-11-Gist_integration.md

```diff
@@ -0,0 +1,40 @@
+# Gist Support
+
+In the [Plutus Playground](https://prod.playground.plutus.iohkdev.io/) you can now save your contracts and simulations to Github [gists](https://help.github.com/en/articles/about-gists) and read them from there. 
+
+## Getting started
+
+To use the gist integration you need to create an account on [Github](https://github.com) if you don't already have one.
+
+Open the Playground and click on the "Login" button in the top right corner:
+
+![Gist login button](./img/2019-04-11-gist-login.PNG)

```

Comment:
IMO screenshots are unnecessary and the explanation is fine, but I have a thing against committing screenshots :p